cli/args/flags.rs

```diff
@@ -3090,6 +3067,7 @@ fn lock_arg() -> Arg {
 
 If value is not provided, defaults to \"deno.lock\" in the current working directory.")
     .num_args(0..=1)
+    // todo(dsherret): remove value_parser!(PathBuf) and instead parse as string
```

Comment:
Can you opean an issue about it?

Reply:
I opened https://github.com/denoland/deno/issues/21954